db/migrate/20160308165211_move_network_port_cloud_subnet_id_to_network_ports_cloud_subnets.rb

```diff
@@ -0,0 +1,34 @@
+class MoveNetworkPortCloudSubnetIdToNetworkPortsCloudSubnets < ActiveRecord::Migration[5.0]
+  class NetworkPort < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class CloudSubnet < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class CloudSubnetNetworkPort < ActiveRecord::Base
+    self.table_name = "cloud_subnets_network_ports"
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    # Move NetworkPort belongs_to :cloud_subnet to NetworkPort has_many :cloud_subnets, :through => :cloud_subnet_network_port
+    NetworkPort.all.each do |network_port|
```

Comment:
not sure how many can be theoretically there, but if there are many, maybe use `find_each` to use batches?